Meta/serenity.sh

```diff
@@ -91,7 +91,7 @@ fi
 if [ -n "$1" ]; then
     TARGET="$1"; shift
 else
-    TARGET="${SERENITY_ARCH:-"i686"}"
+    TARGET="${SERENITY_ARCH:-"x86_64"}"
```

Comment:
This default is defined in more places than just here. At the very least Ports has their own in `.hosted_defs.sh`.

Reply:
Done.